mantidimaging/gui/windows/recon/presenter.py

```diff
@@ -439,8 +447,12 @@ def _completed_finding_cors(task: TaskWorkerThread) -> None:
                               },
                               tracker=self.async_tracker)
 
-    def proj_180_degree_shape_matches_images(self, images) -> bool:
-        return self.model.proj_180_degree_shape_matches_images(images)
+    def proj_180_degree_shape_matches_images(self, images: ImageStack) -> bool:
+        result = self.model.proj_180_degree_shape_matches_images(images)
+        if isinstance(result, bool):
+            return result
+        else:
+            raise ValueError("Expected a boolean from proj_180_degree_shape_matches_images")
```

Comment:
This change should not be needed once `model.proj_180_degree_shape_matches_images()` has its return type fixed.

Reply:
This can be changed back to the original version